compute_tools/src/bin/compute_ctl.rs

```diff
@@ -31,7 +31,7 @@
 //!             -C 'postgresql://cloud_admin@localhost/postgres' \
 //!             -S /var/db/postgres/specs/current.json \
 //!             -b /usr/local/bin/postgres \
-//!             -r {"bucket": "neon-dev-extensions-eu-central-1", "region": "eu-central-1"}
```

Comment:
Do you need to make changes in the control plane? Maybe, it'll be less risky to add a new option and deprecate existing one?

Reply:
or, I saw the hack. Nevermind.